uisdk/src/main/java/com/karhoo/uisdk/util/extension/ServiceCancellationExt.kt

```diff
@@ -8,21 +8,25 @@ import com.karhoo.uisdk.util.CANCELLATION_BEFORE_DRIVER_EN_ROUTE
 import com.karhoo.uisdk.util.CANCELLATION_TIME_BEFORE_PICKUP
 
 fun ServiceCancellation.getCancellationText(context: Context): String? {
-    if (minutes == 0) {
-        return null
-    }
+    var text: String?
```

Comment:
Do we need a unit test class for the extension functions?

Reply:
Don't think so, not seeing any for the other extensions. The unit tests of the views covers all the scenarios/branches of the extension

Reply:
I think we probably should look at covering the extensions. They might get tested in the other Views but really if there's logic in there, then shouldn't it be covered by tests rather than in the tests for a different class?

Reply:
Okay, shall we add the tests during this ticket or create a separate one?

Reply:
Separate one?